third_party/blink/renderer/bindings/core/v8/record_replay_events.cc

```diff
@@ -85,33 +92,22 @@ UserEventProbe::UserEventProbe(const char* name,
     : name_(name ? String(name) : atomic_name),
       event_target_(event_target),
       is_callback_(is_callback) {
-  // Disabled by default, see https://linear.app/replay/issue/RUN-1251
-  if (IsRecordingOrReplaying() &&
-      !FeatureEnabled("disable-collect-events") &&
-      !AreEventsDisallowed()) {
-    ReplayNotifyBeforeEvent(name_, event_target_, is_callback);
-  }
+  ReplayNotifyBeforeEvent(name_, event_target_, is_callback);
 }
 
 UserEventProbe::UserEventProbe(const char* name,
                                const AtomicString& atomic_name,
-                               EventTarget* event_target) : 
-                               UserEventProbe(name, atomic_name, event_target, !event_target)
-{ }
+                               EventTarget* event_target)
+    : UserEventProbe(name, atomic_name, event_target, !event_target) {}
 
+// NOTE: This is used for capturing script run events but it does not work.
 UserEventProbe::UserEventProbe()
-    // NOTE: This is used for capturing script run events but it does not work.
-    // TODO: uncomment - https://linear.app/replay/issue/RUN-1271
-    // UserEventProbe("scriptFirstStatement", AtomicString(), nullptr, false) 
-    {}
+// TODO: uncomment - https://linear.app/replay/issue/RUN-1271
+// UserEventProbe("scriptFirstStatement", AtomicString(), nullptr, false)
```

Comment:
rm

Reply:
This code should be in as-is. It cannot be different from this. I left a TODO as to what we have to fix to make it work.

Reply:
I'll just remove it, although i don't agree with the removal 🤔